### PR TITLE
Add ember module

### PR DESCRIPTION
**Pull request by ptcNOP**
Note: `lief` does not support python3.4...do we still need to support python3.4?

### CODE REVIEW

**Comment by awest1339** [comment]
@Drewsif thoughts on dropping python 3.4 support?

File: modules/MachineLearning/EndgameEmber.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    except lgb.LightGBMError as e:
+        print("Unable to load model, {}. ({})".format(conf['model'], e))
+        return False
+
+    return True
+
+
+def scan(filelist, conf=DEFAULTCONF):
+    results = []
+    model_path = os.path.join(os.path.realpath(os.path.dirname(sys.argv[0])), 'etc', conf['model'])
```
Comment (ptcNOP) [inline modules/MachineLearning/EndgameEmber.py]: Don't reinitialize for each scan.

**Comment by ptcNOP** [comment]
We talked about removing the Ember model from VCS and providing instructions for downloading/updating. This probably isn't a good idea since the model is part of a 1.6GB download that expands out to 9.2GB.

**Comment by awest1339** [comment]
Ok. I'm fine w/ this approach then

**Pull request merged**